Versions/Common/mounting.class.lua

```diff
@@ -29,8 +29,14 @@ function BeStride:Druid()
 		return BeStride_Mount:DruidAquaticForm()
 	elseif self:MovementCheck() and IsOutdoors() then
 		return BeStride_Mount:DruidTravelForm()
-	elseif GetShapeshiftForm() == 3 then
-		return BeStride_Mount:MountSpell(SpellToName(783))
+	elseif BeStride_Game == "Mainline" and GetShapeshiftForm() == 3 then
+		return BeStride_Mount:DruidTravelForm()
+	elseif BeStride_Game == "Wrath" and GetShapeshiftForm() == 2 then
+		return BeStride_Mount:DruidTravelForm()
+	elseif BeStride_Game == "Wrath" and GetShapeshiftForm() == 4 then
+		return BeStride_Mount:DruidAquaticForm()
+	elseif BeStride_Game == "Wrath" and GetShapeshiftForm() == 6 then
+		return BeStride_Mount:DruidFlightForm()
```

Comment:
I'm not happy to add all these conditions, maybe I could move this logic somewhere else, But what I understood here is this case handle removing the current form on Mainline, so each form must be checked separately on Wrath.